packages/aws-cdk-lib/aws-ecr/lib/repository.ts

```diff
@@ -1,5 +1,8 @@
 import { EOL } from 'os';
 import * as path from 'path';
+import { IConstruct, Construct } from 'constructs';
+import { CfnRepository } from './ecr.generated';
+import { LifecycleRule, TagStatus } from './lifecycle';
```

Comment:
Wait, why these imports?

Reply:
Ohh, these just are moved up by eslint. I believe Momo recently fixed all of these after repo restructure.